src/test/java/com/gooddata/dataload/processes/ProcessServiceIT.java

```diff
@@ -60,7 +61,7 @@ public void shouldCreateProcess() throws Exception {
         onRequest()
                 .havingMethodEqualTo("POST")
                 .havingPathEqualTo(PROCESSES_PATH)
-            .respond()
+                .respond()
```

Comment:
actually those indents are intentional for better readability, please leave the there